.github/workflows/commit.yml

```diff
@@ -114,33 +116,43 @@ jobs:
       - name: Setup Python
         uses: actions/setup-python@v4
         with:
-          python-version: 3.8
+          python-version: ${{ env.PYTHON_VERSION }}
           cache: 'pip'
           cache-dependency-path: pyproject.toml
 
       - name: Install Python dependencies
         run: |
           pip install --upgrade pip
           pip install .
-          pip install ".[test, optional]"
+          pip install ".[test,optional]"
+      
+      - name: Upload removed dependencies log
+        uses: actions/upload-artifact@v3
+        with:
+          name: smoke-test-removed-dependencies
+          path: ./removed_dependencies.txt
```

Comment:
This doesn't seem to be needed anymore, and it shows a warning.

Reply:
thanks, removed.